src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -3838,6 +3872,39 @@ namespace winrt::TerminalApp::implementation
         CATCH_RETURN()
     }
 
+    TerminalApp::IPaneContent TerminalPage::_makeSettingsPane()
```

Comment:
not actually making the pane containing settings; it's making the content FOR the settings pane.